db/migrate/20250509154018_add_type_to_job_applications.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToJobApplications < ActiveRecord::Migration[7.0]
+  def change
+    add_column :job_applications, :type, :string
```

Comment:
amazed it lets you do this - when I added the type field to Vacancy, it insisted that I create a PR first to ignore the type column (so that this migration doesn't crash running instances which don't have the 2 types (yet)